### PR TITLE
update the documentation of two function in scipy.special

### PR DESCRIPTION
**Pull request by mherkazandjian**
- i converted some formulas to latex form
- added parameter description

### CODE REVIEW

File: scipy/special/_ufuncs.pyx [hunk 1]
```diff
@@ -6538,7 +6538,11 @@ cdef char ufunc_gamma_types[8]
 cdef char *ufunc_gamma_doc = (
     "gamma(z)\n"
     "\n"
-    "Gamma function.\n"
+    "Gamma function\n"
+    "\n"
+    ".. math::\n"
+    "\n"
+    "    \Gamma(z) = \int_0^\infty x^{z-1} e^{-x} dx = (z - 1)!\n"
```
Comment (pv) [inline scipy/special/_ufuncs.pyx]: These files shouldn't be edited manually, instead add the formula to `add_newdocs.py` (this formula is missing from there), and run `generate_ufuncs.py`.

File: scipy/special/add_newdocs.py [hunk 2]
```diff
@@ -5734,18 +5734,34 @@ def add_newdoc(place, name, doc):
...
+    .. math::
 
-        (z)_m = gamma(z + m) / gamma(z)
+        (z)_m = \frac{\Gamma(z + m)}{\Gamma(z)}
 
     For positive integer `m` it reads::
 
+    .. math::
+
         (z)_m = z * (z + 1) * ... * (z + m - 1)
```
Comment (pv) [inline scipy/special/add_newdocs.py]: Typically you don't write `*` for multiplication in math formulas.

**Review by pv — CHANGES_REQUESTED** [review]

File: scipy/special/_ufuncs.pyx (hunk 1 above)
Comment (mherkazandjian) [inline scipy/special/_ufuncs.pyx]: oh ok. thnx for the tip. i noticed the duplication but didn't know that there a script that does that.

File: scipy/special/add_newdocs.py (hunk 2 above)
Comment (mherkazandjian) [inline scipy/special/add_newdocs.py]: done

File: scipy/special/cython_special.pyx [hunk 3]
```diff
@@ -994,50 +994,50 @@ include "_cython_special.pxi"
 from ._agm cimport agm as _func_agm
 ctypedef double _proto_agm_t(double, double) nogil
 cdef _proto_agm_t *_proto_agm_t_var = &_func_agm
-cdef extern from "_ufuncs_defs.h":
+cdef extern from "./_ufuncs_defs.h":
```
Comment (pv) [inline scipy/special/cython_special.pyx]: Could you run `python generate_ufuncs.py` to avoid adding the `./`.

File: scipy/special/add_newdocs.py [hunk 4]
```diff
@@ -2619,6 +2619,10 @@ def add_newdoc(place, name, doc):
 
     Gamma function.
 
+    .. math::
+
+          \Gamma(z) = \int_0^\infty x^{z-1} e^{-x} dx = (z - 1)!
```
Comment (pv) [inline scipy/special/add_newdocs.py]: Need to change it to `r"""` above

File: scipy/special/add_newdocs.py [hunk 5]
```diff
@@ -5734,18 +5738,34 @@ def add_newdoc(place, name, doc):
...
 
-    The Pochhammer symbol (rising factorial), is defined as::
+    The Pochhammer symbol (rising factorial), is defined as
 
-        (z)_m = gamma(z + m) / gamma(z)
+    .. math::
+
+        (z)_m = \frac{\Gamma(z + m)}{\Gamma(z)}
 
     For positive integer `m` it reads::
```
Comment (pv) [inline scipy/special/add_newdocs.py]: remove ::

**Comment by mherkazandjian** [comment]
btw, is it mentioned somewhere in the dev guides that one should run ````python generate_ufuncs.py```` to 
generate the docs?

moreover, why is _ufuncs.pyx tracked if it can be generated automatically?

**Pull request merged**